libvast/src/plugin.cpp

```diff
@@ -245,7 +242,7 @@ load(const std::vector<std::string>& bundled_plugins,
   // Sort loaded plugins by name.
   std::sort(get_mutable().begin(), get_mutable().end(),
             [](const auto& lhs, const auto& rhs) {
-              return std::strcmp(lhs->name(), rhs->name()) < 0;
+              return lhs->name() < rhs->name();
```

Comment:
Isn't this [`lexicographical_compare`](https://en.cppreference.com/w/cpp/algorithm/lexicographical_compare)?

Reply:
Same deal as `std::string_view::operator<=>(std::string_view)`, just less readable.